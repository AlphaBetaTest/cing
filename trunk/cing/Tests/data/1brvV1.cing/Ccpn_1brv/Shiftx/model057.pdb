REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       3.411   2.318   5.824  1.00  0.00              
ATOM      2  CA  VAL A 171       3.668   2.066   4.374  1.00  0.00              
ATOM      3  HA  VAL A 171       3.664   3.055   3.939  1.00  0.00              
ATOM      4  CB  VAL A 171       5.076   1.419   4.087  1.00  0.00              
ATOM      5  HB  VAL A 171       4.923   0.327   4.041  1.00  0.00              
ATOM      6  CG1 VAL A 171       5.630   1.859   2.718  1.00  0.00              
ATOM      7 HG11 VAL A 171       4.864   1.811   1.927  1.00  0.00              
ATOM      8 HG12 VAL A 171       5.984   2.906   2.733  1.00  0.00              
ATOM      9 HG13 VAL A 171       6.474   1.231   2.373  1.00  0.00              
ATOM     10  CG2 VAL A 171       6.221   1.659   5.100  1.00  0.00              
ATOM     11 HG21 VAL A 171       6.447   2.735   5.219  1.00  0.00              
ATOM     12 HG22 VAL A 171       5.964   1.266   6.101  1.00  0.00              
ATOM     13 HG23 VAL A 171       7.157   1.155   4.796  1.00  0.00              
ATOM     14  C   VAL A 171       2.534   1.337   3.559  1.00  0.00              
ATOM     15  O   VAL A 171       2.225   1.835   2.476  1.00  0.00              
ATOM     16  N   PRO A 172       1.926   0.187   3.973  1.00  0.00              
ATOM     17  CD  PRO A 172       2.171  -0.379   5.296  1.00  0.00              
ATOM     18  CA  PRO A 172       0.908  -0.562   3.185  1.00  0.00              
ATOM     19  HA  PRO A 172       1.458  -1.095   2.384  1.00  0.00              
ATOM     20  CB  PRO A 172       0.415  -1.604   4.216  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.472  -1.232   4.769  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.115  -2.557   3.742  1.00  0.00              
ATOM     23  CG  PRO A 172       1.575  -1.780   5.198  1.00  0.00              
ATOM     24  HG2 PRO A 172       1.254  -2.171   6.182  1.00  0.00              
ATOM     25  HG3 PRO A 172       2.323  -2.488   4.792  1.00  0.00              
ATOM     26  HD2 PRO A 172       1.659   0.286   6.023  1.00  0.00              
ATOM     27  HD3 PRO A 172       3.253  -0.365   5.534  1.00  0.00              
ATOM     28  C   PRO A 172      -0.269   0.247   2.559  1.00  0.00              
ATOM     29  O   PRO A 172      -0.890   1.084   3.217  1.00  0.00              
ATOM     30  N   CYS A 173      -0.596  -0.044   1.295  1.00  0.00              
ATOM     31  H   CYS A 173       0.029  -0.692   0.816  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.649   0.671   0.526  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.409   1.733   0.490  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.469   0.254  -0.925  1.00  0.00              
ATOM     35  HB2 CYS A 173      -0.791  -0.588  -1.077  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.412  -0.063  -1.361  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.643   1.527  -1.864  1.00  0.00              
ATOM     38  C   CYS A 173      -3.093   0.631   1.117  1.00  0.00              
ATOM     39  O   CYS A 173      -3.784   1.655   1.092  1.00  0.00              
ATOM     40  N   SER A 174      -3.503  -0.497   1.726  1.00  0.00              
ATOM     41  H   SER A 174      -2.819  -1.259   1.715  1.00  0.00              
ATOM     42  CA  SER A 174      -4.729  -0.541   2.578  1.00  0.00              
ATOM     43  HA  SER A 174      -5.568  -0.217   1.957  1.00  0.00              
ATOM     44  CB  SER A 174      -4.967  -2.003   3.016  1.00  0.00              
ATOM     45  HB2 SER A 174      -5.932  -2.066   3.560  1.00  0.00              
ATOM     46  HB3 SER A 174      -5.079  -2.651   2.130  1.00  0.00              
ATOM     47  OG  SER A 174      -3.917  -2.505   3.853  1.00  0.00              
ATOM     48  HG  SER A 174      -3.982  -2.041   4.696  1.00  0.00              
ATOM     49  C   SER A 174      -4.771   0.401   3.834  1.00  0.00              
ATOM     50  O   SER A 174      -5.831   0.601   4.430  1.00  0.00              
ATOM     51  N   THR A 175      -3.615   0.938   4.244  1.00  0.00              
ATOM     52  H   THR A 175      -2.802   0.655   3.680  1.00  0.00              
ATOM     53  CA  THR A 175      -3.472   1.901   5.372  1.00  0.00              
ATOM     54  HA  THR A 175      -4.443   1.946   5.901  1.00  0.00              
ATOM     55  CB  THR A 175      -2.443   1.337   6.402  1.00  0.00              
ATOM     56  HB  THR A 175      -2.474   1.993   7.297  1.00  0.00              
ATOM     57  OG1 THR A 175      -1.106   1.350   5.903  1.00  0.00              
ATOM     58  HG1 THR A 175      -1.145   1.293   4.937  1.00  0.00              
ATOM     59  CG2 THR A 175      -2.750  -0.103   6.863  1.00  0.00              
ATOM     60 HG21 THR A 175      -3.800  -0.200   7.201  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.622  -0.824   6.033  1.00  0.00              
ATOM     62 HG23 THR A 175      -2.087  -0.425   7.680  1.00  0.00              
ATOM     63  C   THR A 175      -3.170   3.382   4.921  1.00  0.00              
ATOM     64  O   THR A 175      -3.429   4.307   5.691  1.00  0.00              
ATOM     65  N   CYS A 176      -2.637   3.627   3.699  1.00  0.00              
ATOM     66  H   CYS A 176      -2.680   2.841   3.046  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.463   4.982   3.115  1.00  0.00              
ATOM     68  HA  CYS A 176      -1.982   5.649   3.857  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.479   4.737   1.946  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.842   4.043   1.196  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.295   5.618   1.308  1.00  0.00              
ATOM     72  SG  CYS A 176       0.105   4.164   2.587  1.00  0.00              
ATOM     73  C   CYS A 176      -3.829   5.649   2.719  1.00  0.00              
ATOM     74  O   CYS A 176      -4.031   6.839   2.962  1.00  0.00              
ATOM     75  N   GLU A 177      -4.739   4.876   2.092  1.00  0.00              
ATOM     76  H   GLU A 177      -4.362   3.991   1.744  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.155   5.242   1.812  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.458   4.505   1.056  1.00  0.00              
ATOM     79  CB  GLU A 177      -7.022   4.989   3.088  1.00  0.00              
ATOM     80  HB2 GLU A 177      -6.641   5.633   3.905  1.00  0.00              
ATOM     81  HB3 GLU A 177      -8.052   5.358   2.914  1.00  0.00              
ATOM     82  CG  GLU A 177      -7.123   3.527   3.602  1.00  0.00              
ATOM     83  HG2 GLU A 177      -6.116   3.137   3.832  1.00  0.00              
ATOM     84  HG3 GLU A 177      -7.655   3.510   4.570  1.00  0.00              
ATOM     85  CD  GLU A 177      -7.812   2.547   2.655  1.00  0.00              
ATOM     86  OE1 GLU A 177      -7.232   1.914   1.779  1.00  0.00              
ATOM     87  OE2 GLU A 177      -9.150   2.462   2.881  1.00  0.00              
ATOM     88  C   GLU A 177      -6.452   6.600   1.112  1.00  0.00              
ATOM     89  O   GLU A 177      -7.312   7.390   1.514  1.00  0.00              
ATOM     90  N   GLY A 178      -5.749   6.810  -0.001  1.00  0.00              
ATOM     91  H   GLY A 178      -4.855   6.309   0.035  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.858   8.059  -0.781  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.913   7.750  -1.835  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.806   8.565  -0.571  1.00  0.00              
ATOM     95  C   GLY A 178      -4.719   9.092  -0.661  1.00  0.00              
ATOM     96  O   GLY A 178      -4.748  10.098  -1.372  1.00  0.00              
ATOM     97  N   ASN A 179      -3.707   8.857   0.193  1.00  0.00              
ATOM     98  H   ASN A 179      -3.646   7.879   0.495  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.517   9.720   0.277  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.759  10.753  -0.041  1.00  0.00              
ATOM    101  CB  ASN A 179      -1.882   9.604   1.676  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.632   9.442   2.476  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.250   8.691   1.629  1.00  0.00              
ATOM    104  CG  ASN A 179      -0.964  10.765   2.019  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.196  10.811   1.627  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.459  11.748   2.718  1.00  0.00              
ATOM    107 HD21 ASN A 179      -2.428  11.641   3.030  1.00  0.00              
ATOM    108 HD22 ASN A 179      -0.793  12.496   2.916  1.00  0.00              
ATOM    109  C   ASN A 179      -1.536   9.181  -0.750  1.00  0.00              
ATOM    110  O   ASN A 179      -0.890   8.143  -0.568  1.00  0.00              
ATOM    111  N   LEU A 180      -1.415   9.910  -1.836  1.00  0.00              
ATOM    112  H   LEU A 180      -2.052  10.707  -1.922  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.623   9.425  -2.960  1.00  0.00              
ATOM    114  HA  LEU A 180      -1.042   8.412  -3.031  1.00  0.00              
ATOM    115  CB  LEU A 180      -1.034  10.352  -4.114  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.199  11.384  -3.721  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.189  10.404  -4.770  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.221   9.917  -4.982  1.00  0.00              
ATOM    119  HG  LEU A 180      -3.022   9.765  -4.261  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.632  11.030  -5.958  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.868  11.976  -5.436  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.834  11.253  -6.693  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.536  10.755  -6.534  1.00  0.00              
ATOM    124  CD2 LEU A 180      -1.977   8.616  -5.767  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.741   7.761  -5.107  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.866   8.319  -6.355  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.135   8.713  -6.479  1.00  0.00              
ATOM    128  C   LEU A 180       0.912   9.208  -2.763  1.00  0.00              
ATOM    129  O   LEU A 180       1.518   8.377  -3.433  1.00  0.00              
ATOM    130  N   ALA A 181       1.490   9.926  -1.803  1.00  0.00              
ATOM    131  H   ALA A 181       0.772  10.480  -1.337  1.00  0.00              
ATOM    132  CA  ALA A 181       2.849   9.663  -1.287  1.00  0.00              
ATOM    133  HA  ALA A 181       3.524   9.585  -2.152  1.00  0.00              
ATOM    134  CB  ALA A 181       3.257  10.892  -0.469  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.240  11.810  -1.082  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.588  11.045   0.399  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.282  10.769  -0.076  1.00  0.00              
ATOM    138  C   ALA A 181       2.931   8.317  -0.481  1.00  0.00              
ATOM    139  O   ALA A 181       3.744   7.464  -0.828  1.00  0.00              
ATOM    140  N   CYS A 182       2.066   8.098   0.536  1.00  0.00              
ATOM    141  H   CYS A 182       1.250   8.720   0.488  1.00  0.00              
ATOM    142  CA  CYS A 182       1.942   6.794   1.263  1.00  0.00              
ATOM    143  HA  CYS A 182       2.817   6.539   1.820  1.00  0.00              
ATOM    144  CB  CYS A 182       0.916   7.054   2.362  1.00  0.00              
ATOM    145  HB2 CYS A 182       1.083   8.017   2.855  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.076   7.131   1.922  1.00  0.00              
ATOM    147  SG  CYS A 182       0.867   5.717   3.564  1.00  0.00              
ATOM    148  C   CYS A 182       1.692   5.557   0.351  1.00  0.00              
ATOM    149  O   CYS A 182       2.399   4.549   0.402  1.00  0.00              
ATOM    150  N   LEU A 183       0.697   5.708  -0.512  1.00  0.00              
ATOM    151  H   LEU A 183       0.196   6.602  -0.424  1.00  0.00              
ATOM    152  CA  LEU A 183       0.369   4.776  -1.597  1.00  0.00              
ATOM    153  HA  LEU A 183       0.066   3.811  -1.159  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.828   5.486  -2.258  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.531   6.509  -2.532  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.987   5.041  -3.198  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.211   5.475  -1.605  1.00  0.00              
ATOM    158  HG  LEU A 183      -2.050   5.988  -0.641  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.185   6.251  -2.508  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.155   5.887  -3.554  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.231   6.151  -2.181  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.951   7.329  -2.541  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.773   4.052  -1.434  1.00  0.00              
ATOM    164 HD21 LEU A 183      -2.111   3.413  -0.834  1.00  0.00              
ATOM    165 HD22 LEU A 183      -3.755   4.042  -0.931  1.00  0.00              
ATOM    166 HD23 LEU A 183      -2.895   3.532  -2.405  1.00  0.00              
ATOM    167  C   LEU A 183       1.500   4.494  -2.650  1.00  0.00              
ATOM    168  O   LEU A 183       1.585   3.381  -3.160  1.00  0.00              
ATOM    169  N   SER A 184       2.332   5.489  -2.997  1.00  0.00              
ATOM    170  H   SER A 184       2.133   6.341  -2.453  1.00  0.00              
ATOM    171  CA  SER A 184       3.528   5.278  -3.878  1.00  0.00              
ATOM    172  HA  SER A 184       3.204   4.695  -4.764  1.00  0.00              
ATOM    173  CB  SER A 184       4.026   6.640  -4.405  1.00  0.00              
ATOM    174  HB2 SER A 184       3.198   7.193  -4.890  1.00  0.00              
ATOM    175  HB3 SER A 184       4.371   7.282  -3.570  1.00  0.00              
ATOM    176  OG  SER A 184       5.079   6.468  -5.358  1.00  0.00              
ATOM    177  HG  SER A 184       5.558   5.666  -5.112  1.00  0.00              
ATOM    178  C   SER A 184       4.691   4.446  -3.224  1.00  0.00              
ATOM    179  O   SER A 184       5.393   3.720  -3.932  1.00  0.00              
ATOM    180  N   LEU A 185       4.924   4.580  -1.906  1.00  0.00              
ATOM    181  H   LEU A 185       4.300   5.256  -1.439  1.00  0.00              
ATOM    182  CA  LEU A 185       5.886   3.711  -1.145  1.00  0.00              
ATOM    183  HA  LEU A 185       6.807   3.556  -1.707  1.00  0.00              
ATOM    184  CB  LEU A 185       6.199   4.371   0.235  1.00  0.00              
ATOM    185  HB2 LEU A 185       5.315   4.212   0.884  1.00  0.00              
ATOM    186  HB3 LEU A 185       7.016   3.801   0.717  1.00  0.00              
ATOM    187  CG  LEU A 185       6.537   5.876   0.314  1.00  0.00              
ATOM    188  HG  LEU A 185       5.653   6.406  -0.091  1.00  0.00              
ATOM    189  CD1 LEU A 185       6.717   6.321   1.774  1.00  0.00              
ATOM    190 HD11 LEU A 185       5.816   6.106   2.380  1.00  0.00              
ATOM    191 HD12 LEU A 185       7.571   5.816   2.262  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.893   7.410   1.848  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.782   6.253  -0.501  1.00  0.00              
ATOM    194 HD21 LEU A 185       7.651   6.015  -1.573  1.00  0.00              
ATOM    195 HD22 LEU A 185       7.996   7.337  -0.445  1.00  0.00              
ATOM    196 HD23 LEU A 185       8.685   5.718  -0.154  1.00  0.00              
ATOM    197  C   LEU A 185       5.458   2.234  -0.892  1.00  0.00              
ATOM    198  O   LEU A 185       6.297   1.372  -0.612  1.00  0.00              
ATOM    199  N   CYS A 186       4.156   1.966  -0.956  1.00  0.00              
ATOM    200  H   CYS A 186       3.596   2.811  -0.818  1.00  0.00              
ATOM    201  CA  CYS A 186       3.581   0.699  -0.561  1.00  0.00              
ATOM    202  HA  CYS A 186       4.051   0.516   0.422  1.00  0.00              
ATOM    203  CB  CYS A 186       2.087   1.106  -0.435  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.665   0.445   0.320  1.00  0.00              
ATOM    205  HB3 CYS A 186       1.840   2.103  -0.048  1.00  0.00              
ATOM    206  SG  CYS A 186       1.240   0.898  -2.015  1.00  0.00              
ATOM    207  C   CYS A 186       3.737  -0.607  -1.419  1.00  0.00              
ATOM    208  O   CYS A 186       4.335  -0.694  -2.490  1.00  0.00              
ATOM    209  N   HIS A 187       3.031  -1.609  -0.891  1.00  0.00              
ATOM    210  H   HIS A 187       3.010  -1.479   0.128  1.00  0.00              
ATOM    211  CA  HIS A 187       2.873  -2.962  -1.488  1.00  0.00              
ATOM    212  HA  HIS A 187       3.751  -3.220  -2.100  1.00  0.00              
ATOM    213  CB  HIS A 187       2.801  -3.995  -0.326  1.00  0.00              
ATOM    214  HB2 HIS A 187       1.814  -3.962   0.123  1.00  0.00              
ATOM    215  HB3 HIS A 187       2.847  -4.997  -0.768  1.00  0.00              
ATOM    216  CG  HIS A 187       3.787  -3.887   0.833  1.00  0.00              
ATOM    217  ND1 HIS A 187       3.615  -2.997   1.884  1.00  0.00              
ATOM    218  CD2 HIS A 187       5.089  -4.360   0.782  1.00  0.00              
ATOM    219  CE1 HIS A 187       4.895  -2.987   2.364  1.00  0.00              
ATOM    220  NE2 HIS A 187       5.846  -3.794   1.795  1.00  0.00              
ATOM    221  HD2 HIS A 187       5.431  -4.942  -0.055  1.00  0.00              
ATOM    222  HE1 HIS A 187       5.166  -2.219   3.074  1.00  0.00              
ATOM    223  C   HIS A 187       1.585  -3.100  -2.392  1.00  0.00              
ATOM    224  O   HIS A 187       1.639  -3.722  -3.454  1.00  0.00              
ATOM    225  N   ILE A 188       0.442  -2.507  -1.988  1.00  0.00              
ATOM    226  H   ILE A 188       0.540  -2.169  -1.029  1.00  0.00              
ATOM    227  CA  ILE A 188      -0.869  -2.585  -2.727  1.00  0.00              
ATOM    228  HA  ILE A 188      -0.836  -3.383  -3.403  1.00  0.00              
ATOM    229  CB  ILE A 188      -2.049  -3.041  -1.773  1.00  0.00              
ATOM    230  HB  ILE A 188      -2.253  -2.223  -1.071  1.00  0.00              
ATOM    231  CG2 ILE A 188      -3.386  -3.291  -2.525  1.00  0.00              
ATOM    232 HG21 ILE A 188      -3.290  -4.072  -3.302  1.00  0.00              
ATOM    233 HG22 ILE A 188      -4.197  -3.610  -1.844  1.00  0.00              
ATOM    234 HG23 ILE A 188      -3.760  -2.382  -3.031  1.00  0.00              
ATOM    235  CG1 ILE A 188      -1.761  -4.285  -0.871  1.00  0.00              
ATOM    236 HG12 ILE A 188      -0.959  -4.031  -0.153  1.00  0.00              
ATOM    237 HG13 ILE A 188      -2.638  -4.483  -0.222  1.00  0.00              
ATOM    238  CD1 ILE A 188      -1.370  -5.594  -1.588  1.00  0.00              
ATOM    239 HD11 ILE A 188      -0.451  -5.480  -2.194  1.00  0.00              
ATOM    240 HD12 ILE A 188      -1.179  -6.408  -0.864  1.00  0.00              
ATOM    241 HD13 ILE A 188      -2.168  -5.949  -2.267  1.00  0.00              
ATOM    242  C   ILE A 188      -1.104  -1.374  -3.718  1.00  0.00              
ATOM    243  O   ILE A 188      -2.112  -0.667  -3.675  1.00  0.00              
ATOM    244  N   GLU A 189      -0.182  -1.184  -4.685  1.00  0.00              
ATOM    245  H   GLU A 189       0.663  -1.746  -4.570  1.00  0.00              
ATOM    246  CA  GLU A 189      -0.226  -0.058  -5.658  1.00  0.00              
ATOM    247  HA  GLU A 189      -0.535   0.847  -5.106  1.00  0.00              
ATOM    248  CB  GLU A 189       1.200   0.221  -6.216  1.00  0.00              
ATOM    249  HB2 GLU A 189       1.577  -0.661  -6.772  1.00  0.00              
ATOM    250  HB3 GLU A 189       1.116   1.021  -6.979  1.00  0.00              
ATOM    251  CG  GLU A 189       2.256   0.668  -5.168  1.00  0.00              
ATOM    252  HG2 GLU A 189       1.793   1.380  -4.470  1.00  0.00              
ATOM    253  HG3 GLU A 189       2.567  -0.187  -4.538  1.00  0.00              
ATOM    254  CD  GLU A 189       3.493   1.380  -5.711  1.00  0.00              
ATOM    255  OE1 GLU A 189       3.454   2.436  -6.333  1.00  0.00              
ATOM    256  OE2 GLU A 189       4.647   0.734  -5.397  1.00  0.00              
ATOM    257  C   GLU A 189      -1.267  -0.237  -6.805  1.00  0.00              
TER     258      GLU A 189 
END   
