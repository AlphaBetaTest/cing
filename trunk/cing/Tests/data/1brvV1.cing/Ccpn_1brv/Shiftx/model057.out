# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      120.3954
  171   V  CA      62.2472
  171   V  CB      33.5490
  171   V  C      176.0349
  171   V  H        8.0549
  171   V  HA       4.0480
  171   V  HB       2.1237
  171   V  HG11     0.9651
  171   V  HG12     0.8935
  171   V  HG13     1.1364
  171   V  HG21     0.9386
  171   V  HG22     0.9507
  171   V  HG23     0.9477
  172   P  CA      62.8730
  172   P  CB      31.3831
  172   P  C      176.5602
  172   P  HA       4.1940
  172   P  HB2      2.3168
  172   P  HB3      2.0285
  172   P  HD2      3.6807
  172   P  HD3      3.8171
  172   P  HG2      2.1359
  172   P  HG3      2.2106
  173   C  N      120.6624
  173   C  CA      60.0911
  173   C  CB      40.8854
  173   C  C      176.0475
  173   C  H        8.8917
  173   C  HA       4.3076
  173   C  HB2      3.0222
  173   C  HB3      3.2149
  174   S  N      114.3617
  174   S  CA      60.6711
  174   S  CB      62.5274
  174   S  C      175.3262
  174   S  H        8.1876
  174   S  HA       4.1382
  174   S  HB2      4.0216
  174   S  HB3      4.0095
  175   T  N      111.8106
  175   T  CA      62.7180
  175   T  CB      69.5889
  175   T  C      175.7573
  175   T  H        7.6878
  175   T  HA       4.4261
  175   T  HB       4.2460
  175   T  HG21     1.2146
  175   T  HG22     1.2180
  175   T  HG23     1.2158
  176   C  N      119.4517
  176   C  CA      58.5470
  176   C  CB      40.9460
  176   C  C      173.7377
  176   C  H        8.1410
  176   C  HA       4.3474
  176   C  HB2      2.9919
  176   C  HB3      3.2112
  177   E  N      116.8153
  177   E  CA      57.5868
  177   E  CB      26.9643
  177   E  C      175.4339
  177   E  H        8.9720
  177   E  HA       3.5212
  177   E  HB2      2.1031
  177   E  HB3      1.9807
  177   E  HG2      2.3273
  177   E  HG3      2.3791
  178   G  N      104.1949
  178   G  CA      45.2077
  178   G  C      173.4832
  178   G  H        8.3220
  178   G  HA    -666.0000
  178   G  HA2      3.9605
  178   G  HA3      3.9604
  179   N  N      118.4723
  179   N  CA      52.4397
  179   N  CB      39.3225
  179   N  C      175.2353
  179   N  H        7.8946
  179   N  HA       4.5982
  179   N  HB2      2.7849
  179   N  HB3      2.7328
  179   N  HD21     7.1239
  179   N  HD22     6.5436
  180   L  N      129.1609
  180   L  CA      57.2757
  180   L  CB      41.6158
  180   L  C      179.4150
  180   L  H        8.4599
  180   L  HA       3.9691
  180   L  HB2      1.7246
  180   L  HB3      1.7023
  180   L  HD11     0.9191
  180   L  HD12     0.9188
  180   L  HD13     0.9191
  180   L  HD21     0.2273
  180   L  HD22     1.9510
  180   L  HD23     0.8681
  180   L  HG       1.7199
  181   A  N      120.7088
  181   A  CA      55.2130
  181   A  CB      18.4922
  181   A  C      179.6622
  181   A  H        7.9091
  181   A  HA       3.9008
  181   A  HB1      1.3875
  181   A  HB2      1.3676
  181   A  HB3      1.3337
  182   C  N      116.4907
  182   C  CA      60.1340
  182   C  CB      43.4741
  182   C  C      175.8533
  182   C  H        7.9750
  182   C  HA       4.0954
  182   C  HB2      3.0069
  182   C  HB3      3.2330
  183   L  N      119.0763
  183   L  CA      57.9443
  183   L  CB      41.2211
  183   L  C      179.2710
  183   L  H        7.9344
  183   L  HA       3.8594
  183   L  HB2      1.8633
  183   L  HB3      1.7210
  183   L  HD11     0.9194
  183   L  HD12     0.9917
  183   L  HD13     0.9042
  183   L  HD21     0.3585
  183   L  HD22     2.2528
  183   L  HD23     0.8707
  183   L  HG       1.7031
  184   S  N      112.2625
  184   S  CA      61.2375
  184   S  CB      62.4081
  184   S  C      175.9115
  184   S  H        7.8700
  184   S  HA       4.3794
  184   S  HB2      4.0016
  184   S  HB3      4.0067
  185   L  N      120.8664
  185   L  CA      57.1450
  185   L  CB      41.6392
  185   L  C      174.8761
  185   L  H        7.6151
  185   L  HA       4.1591
  185   L  HB2      1.5919
  185   L  HB3      1.6936
  185   L  HD11     0.9152
  185   L  HD12     0.9144
  185   L  HD13     0.9156
  185   L  HD21     0.3352
  185   L  HD22     2.6546
  185   L  HD23     0.8605
  185   L  HG       1.7969
  186   C  N      112.5033
  186   C  CA      55.1593
  186   C  CB      43.1252
  186   C  C      172.5939
  186   C  H        7.4475
  186   C  HA       4.5416
  186   C  HB2      3.3387
  186   C  HB3      3.3657
  187   H  N      115.1245
  187   H  CA      57.9388
  187   H  CB      29.4695
  187   H  C      176.7595
  187   H  H        8.5276
  187   H  HA       4.3696
  187   H  HB2      3.1402
  187   H  HB3      3.4040
  187   H  HD2      5.8821
  187   H  HE1      6.9885
  188   I  N      123.0541
  188   I  CA      60.7488
  188   I  CB      36.0990
  188   I  C      176.1292
  188   I  H        7.5052
  188   I  HA       4.0579
  188   I  HB       1.8856
  188   I  HD11     0.8861
  188   I  HD12     0.8759
  188   I  HD13     0.8727
  188   I  HG12     1.3068
  188   I  HG13     0.7988
  188   I  HG21     0.9188
  188   I  HG22     0.9184
  188   I  HG23     0.9172
 *189   E  N      119.8256
 *189   E  CA      56.1890
 *189   E  CB      30.2502
 *189   E  C      177.6092
 *189   E  H        7.7641
 *189   E  HA       4.1489
 *189   E  HB2      1.7275
 *189   E  HB3      1.7577
 *189   E  HG2      2.2819
 *189   E  HG3      2.4576
